### PR TITLE
New package: sxyazi.yazi version 0.2.5

### PR DESCRIPTION
**Pull request by MoaidHathot**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/159869)

### CODE REVIEW

**Comment by MoaidHathot** [comment]
Linked Issue:
https://github.com/gg/issues/1088

File: manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 0.2.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: yazi-x86_64-pc-windows-msvc\ya.exe
+  PortableCommandAlias: ya
+- RelativeFilePath: yazi-x86_64-pc-windows-msvc\yazi.exe
+  PortableCommandAlias: yazi
+Installers:
+- Architecture: x64
```
Comment (sxyazi) [inline manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml:14]: Can we also add [Windows aarch64](https://github.com/sxyazi/yazi/releases/download/v0.2.5/yazi-aarch64-pc-windows-msvc.zip) here?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\ya.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\yazi.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 886.)

File: manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml (hunk 1 above)
Comment (MoaidHathot) [reply manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml]: Done :)

**Review by sxyazi — APPROVED** [review]
Thanks, all looks good to me now!

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

File: manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+NestedInstallerType: portable
+Installers:
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: yazi-x86_64-pc-windows-msvc\ya.exe
+    PortableCommandAlias: ya
+  - RelativeFilePath: yazi-x86_64-pc-windows-msvc\yazi.exe
+    PortableCommandAlias: yazi
+  InstallerUrl: https://github.com/sxyazi/yazi/releases/download/v0.2.5/yazi-x86_64-pc-windows-msvc.zip
+  InstallerSha256: CD3B1DC56165AC164354A538B5B3493DE61ED3147E2484DA10C54297946F89A4
```
Comment (stephengillie) [inline manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml:16]:
```suggestion
  InstallerSha256: CD3B1DC56165AC164354A538B5B3493DE61ED3147E2484DA10C54297946F89A4
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- Architecture: arm64
+  NestedInstallerFiles:
+  - RelativeFilePath: yazi-aarch64-pc-windows-msvc\ya.exe
+    PortableCommandAlias: ya
+  - RelativeFilePath: yazi-aarch64-pc-windows-msvc\yazi.exe
+    PortableCommandAlias: yazi
+  InstallerUrl: https://github.com/sxyazi/yazi/releases/download/v0.2.5/yazi-aarch64-pc-windows-msvc.zip
+  InstallerSha256: 9D3E0395B0D66F012A8CB4A7FE97EAD131C89812C12CF2D9C93663811DC12C1E
```
Comment (stephengillie) [inline manifests/s/sxyazi/yazi/0.2.5/sxyazi.yazi.installer.yaml:24]:
```suggestion
  InstallerSha256: 9D3E0395B0D66F012A8CB4A7FE97EAD131C89812C12CF2D9C93663811DC12C1E
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @MoaidHathot,

Manual Validation ended with:
image

**Comment by stephengillie** [comment]
Despite having the dependency installed, this application keeps prompting for the DLL.

**Pull request closed without merging**